interop/cats/src/main/scala/caliban/interop/cats/CatsInterop.scala

```diff
@@ -28,4 +32,20 @@ object CatsInterop {
     Async[F].async { cb =>
       runtime.unsafeRunAsync(graphQL.execute(query))(exit => cb(exit.toEither.void))
     }
+
+  def schema[F[_]: Effect, R, A](implicit ev: Schema[R, A]): Schema[R, F[A]] =
+    new Schema[R, F[A]] {
+      override def toType(isInput: Boolean): __Type =
+        ev.toType(isInput)
+
+      override def resolve(
+        value: F[A],
+        arguments: Map[String, Value]
+      ): ZIO[R, CalibanError.ExecutionError, ResolvedValue] =
+        value.toIO
+          .map(a => ev.resolve(a, arguments))
+          .to[Task]
+          .flatten
+          .refineToOrDie[CalibanError.ExecutionError]
```

Comment:
I'm not completely sure if this is right but we need something like it: ZIO interop only provides an instance of `Task`, which has its error type set to `Throwable`.

Reply:
`ExecutionError` contains an `Option[Throwable]`, so you can do `mapError` just like here https://github.com/ghostdogpr/caliban/blob/master/core/src/main/scala/caliban/schema/Schema.scala#L233 I think?